### PR TITLE
Add merge cronjob

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Related to https://github.com/thoth-station/kebechet/issues/546#issuecomment-851455082

## Does this require new deployment ?

- [ ] Deployment for Test and Stage `AICoE/aicoe-cd` and Prod `operate-first/argocd-apps`.

## Description
New merge cronjob, that aggregates all the data `mi` has collected about kebechet

### CODE REVIEW

File: mi-scheduler/base/cronjob.yaml [hunk 1]
```diff
@@ -133,8 +131,86 @@ spec:
                       key: github-access-token
                 - name: SCHEDULE_KEBECHET_ANALYSIS
                   value: "1"
-                - name: SCHEDULE_GH_REPO_ANALYSIS
+                - name: THOTH_ADJUST_LOGGING
+                  value: 'thoth.mi-scheduler:INFO'
+          restartPolicy: OnFailure
+---
+kind: CronJob
```
Comment (goern) [inline mi-scheduler/base/cronjob.yaml:138]: is this a cronjob or do we want it to be an ArgoCronjobWorkflow? @harshad16

Comment (xtuchyna) [reply mi-scheduler/base/cronjob.yaml:138]: do we want to join kebechet's `merge` and `analyse` into the one ArgoCronjobWorkflow? Or what's the use case here?

**Comment by xtuchyna** [comment]
hello @goern @harshad16 ! :helicopter: 
any news regarding this?

**Comment by xtuchyna** [comment]
aloha @goern @harshad16 any status on this?
@pacospace wdyt?

File: mi-scheduler/base/cronjob.yaml (hunk 1 above)
Comment (goern) [reply mi-scheduler/base/cronjob.yaml:138]: it was more a general question, CronJob is good ;)

**Pull request merged**